openassetio_mediacreation/README.md

```diff
@@ -0,0 +1,11 @@
+# OpenAssetIO-MediaCreation-Package
+
+Openassetio-mediacreation is automatically generated using
+[openassetio-traitgen](https://github.com/OpenAssetIO/OpenAssetIO-TraitGen).
+
+This directory serves as the package directory as far as `setuptools` is
+concerned, despite containing no source files.
+It's a bit of a compromise, as there is no other purpose for this
```

Comment:
[plan] Issue to find a better way.

Reply:
How critical actually is this?

Reply:
It's something that i'd put into a backlog for sure if we ran under a speculative backlog model (ie, have tickets for all things that would be good to do with infinite resources)

But given our conversation about only having items that we actually intend to do, it's not something i'd want to reopen without a reason, assuming this method here continues to function.

Reply:
Why dont we ask @JeanChristopheMorinPerso - and if he has any ideas, but them in the backlog, else park it for now.

Reply:
If it was just us, then I'd say park it, but if we expect other people to create their own repos following this model, then it would be good to smooth the cracks.

Reply:
@feltech Do you think we could merge this right now under the agreement that we're going to consult and plan? I'll make a point to bring it up in standup tommorow so it dosen't get swept under the rug.